source/content/github-actions.md

```diff
@@ -49,7 +49,7 @@ jobs:
     steps:
     - uses: actions/checkout@v4
     - name: Push to Pantheon
-      uses: pantheon/push-to-pantheon@0.5.0
+      uses: pantheon-systems/push-to-pantheon@0.5.0
```

Comment:
@stevector or @jazzsequence should this be updated to the [latest release](https://github.com/pantheon-systems/push-to-pantheon/releases)?

Reply:
reviewed on a call w/Steve and Chris, will update to 0.6.0